### PR TITLE
Home

### PR DESCRIPTION
**Pull request by preethasri**
added modals to homepage
added services for posts
added postslice for feedpost 
added loader in components

### CODE REVIEW

File: src/Components/Aside/Aside.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            <div className="aside-input">
+            <span className="aside-search material-icons">search</span>
+            <input type="text" placeholder="search" />
+            </div>
+            <div className="aside aside-header">
+                <h2>Who To Follow</h2>
+                <div className="aside aside-items">
+                    <div className="aside aside-item">
+                        
+                    <img src="./assets/images/avatar2.jpg" alt="" className="normal-img1 " />
```
Comment (manojasarsa) [inline src/Components/Aside/Aside.js:16]: you can give some name for `alt` attribute here

File: src/Components/Aside/Aside.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                        <div className="aside-item-btn">
+                            <button className="follow-btn">Follow
+
+                            </button>
+
+                        </div>
+
+                    </div>
+                    </div>
+                    {/*  */}
```
Comment (manojasarsa) [inline src/Components/Aside/Aside.js:34]: you can remove this

File: src/Components/Loader/Loader.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.loader-wrapper{
+  background:var(--primary-background-color);
+  margin:auto;
+  height:calc(80%-8.9rem);
+  width:100%;
+
+
+
```
Comment (manojasarsa) [inline src/Components/Loader/Loader.css:8]: you can remove blank lines

File: src/Pages/HomePage/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (manojasarsa) [inline src/Pages/HomePage/HomePage.js:120]: You can remove these empty lines

**Review by manojasarsa — APPROVED** [review]
Your code needs to be structured. 
Use any code formatter for proper spacing and indentation.
Other than that, the code looks clean.

**Review by Shrinkhlarajpoot — APPROVED** [review]
Remove Unwanted Space..and try to make your sidebar with tailwind as well...
Use barrel export for components. Also a suggestion you can include all your slice in a seprate folder Features to make code look well structured.
UI is looking cood...You can merge it.

File: src/Pages/HomePage/HomePage.js (hunk 4 above)
Comment (preethasri) [reply src/Pages/HomePage/HomePage.js:120]: wiil work on this manoj

**Comment by preethasri** [comment]
> 

will work on this shrinkhla

**Pull request merged**